[2024-07-22T02:30:54.028+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-03-20T00:00:00+00:00 map_index=1 [queued]>
[2024-07-22T02:30:54.072+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-03-20T00:00:00+00:00 map_index=1 [queued]>
[2024-07-22T02:30:54.072+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:30:54.072+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-07-22T02:30:54.073+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:30:54.150+0000] {taskinstance.py:1300} INFO - Executing <Mapped(_PythonDecoratedOperator): add_one> on 2022-03-20 00:00:00+00:00
[2024-07-22T02:30:54.154+0000] {standard_task_runner.py:55} INFO - Started process 591 to run task
[2024-07-22T02:30:54.158+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dynamic_task_mapping', 'add_one', 'scheduled__2022-03-20T00:00:00+00:00', '--job-id', '73', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_dynamic_task_mapping.py', '--cfg-path', '/tmp/tmpwbt2m484', '--map-index', '1']
[2024-07-22T02:30:54.166+0000] {standard_task_runner.py:83} INFO - Job 73: Subtask add_one
[2024-07-22T02:30:54.279+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 73 for task add_one ([Errno 2] No such file or directory: '/opt/airflow/logs/dag_id=example_dynamic_task_mapping/run_id=scheduled__2022-03-20T00:00:00+00:00/task_id=add_one/map_index=1/attempt=1.log'; 591)
[2024-07-22T02:30:54.309+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-07-22T02:30:54.391+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
